docs/2.7/ssh_rbac.md

```diff
@@ -105,7 +105,7 @@ spec:
   allow:
     # logins array defines the OS logins a user is allowed to use.
     # a few special variables are supported here (see below)
-    logins: [root, '{{internal.logins}}']
+    logins: [root, '{% raw %}{{internal.logins}}{% endraw %}']
```

Comment:
I guess this is to prevent Jinja from stalling on what it thinks is a substitution - does this string appear in any of the other `ssh_rbac.md` files? Or I guess, why are we changing this particular one in the 2.7 docs but no others?

Reply:
Ow this is was mistake. I've only turned this plugin on 4.2. Going to revert this.